src/current/v23.1/live-migration-service.md

```diff
@@ -0,0 +1,646 @@
+You can use this mode to perform an [immediate cutover](#immediate-cutover).
+{% endcomment %}
+
+## Perform a cutover
+
+### Consistent cutover
+
+A consistent cutover maintains data consistency with [minimal downtime]({% link {{ page.version.version }}/migration-overview.md %}#minimal-downtime). When using the LMS, consistent cutover is handled using the [`molt-lms-cli`](#molt-lms-cli) commands `cutover consistent begin` and `cutover consistent commit`, during which application requests are queued and will be responded to after cutover. This delay in response time is related to the maximum duration of any transactions and queries that need to complete, and the time it takes for replication to catch up from the source to the target database.
+
+{% comment %}
+For more information about the consistent cutover approach, see [Migration Strategy: Live Migration]({% link {{ page.version.version }}/migration-strategy-live-migration.md %}).
+{% endcomment %}
+
+To perform a consistent cutover with the LMS:
+
+{{site.data.alerts.callout_info}}
+These steps assume you have already followed the overall steps to [prepare for migration]({% link {{ page.version.version }}/migration-overview.md %}#prepare-for-migration). In particular, [update your schema and application queries]({% link {{ page.version.version }}/migration-overview.md %}#update-the-schema-and-queries) to work with CockroachDB.
+{{site.data.alerts.end}}
+
+1. [Configure the LMS](#configuration) with your deployment details, and follow our [security recommendations](#security).
+
+1. Set the shadowing mode to [`none`](#none).
+
+	{{site.data.alerts.callout_danger}}
+	Do not use the [`sync`](#sync) or [`strict-sync`](#strict-sync) shadowing modes when performing a consistent cutover. Data correctness and consistency cannot be guaranteed in these configurations.
+	{{site.data.alerts.end}}
+
+1. Set up ongoing replication between the source database and CockroachDB, using a tool such as [change data capture (CDC)]({% link {{ page.version.version }}/cdc-queries.md %}).
+
+1. Send application requests to the LMS, which routes the traffic to the source database. The source database is designated the source of truth.
+
+1. Use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to validate that the replicated data on CockroachDB is consistent with the source of truth.
+
+1. Begin the consistent cutover. **Requests are now queued in the LMS**, including queries from existing connections and new connection requests to the LMS:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt-lms-cli cutover consistent begin {flags}
+	~~~
+
+	This command tells the LMS to pause all application traffic to the source of truth. The LMS then waits for transactions to complete and prepared statements to close.
+
+1. Verify that replication on CockroachDB has caught up with the source of truth. For example, write a marker record to the source database and check that it exists on CockroachDB.
```

Comment:
Can we go into more detail about what a "marker record" is?  I would have no idea what we're talking about here if I were new to migrations.

Reply:
Amending to "insert a row on the source database and check that the row exists on CockroachDB".

Reply:
SGTM